hw/dashboard/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Hardware Development Dashboard"
+---
+
+This page serves as the landing spot for all hardware development within the OpenTitan project.
+
+We start off by providing links to the [summary results of various tool-flows](#toolflows-summary-results) run on all of our [Comportable]({{< relref "doc/rm/comportability_specification" >}}) designs.
+This includes DV simulations, FPV and lint, all of which are run with the `dvsim` tool which serves as the common frontend.
+
+The [Comportable designs dashboard](#comportable-designs-dashboard) following it provides links to their design specifications, DV plans and tracks their current stage of development.
```

Comment:
grammar nerd: should be "links to their design specifications and DV plans, and tracks..."